README.md

```diff
@@ -59,15 +58,6 @@ This repository includes files with plain SQL that can be used to recreate a dat
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍